sale_order_product_recommendation/tests/test_recommendation.py

```diff
@@ -25,7 +25,6 @@ def test_recommendations(self):
                 },
             )
         ]
-        self.new_so.order_line.product_id_change()
```

Comment:
This you can change with: self.new_so.order_line._onchange_product_id_warning() the new way to call the method